stl/inc/type_traits

```diff
@@ -2030,8 +2030,8 @@ struct _Is_swappable : _Is_swappable_with<add_lvalue_reference_t<_Ty>, add_lvalu
 
 // STRUCT TEMPLATE _Swap_cannot_throw
 template <class _Ty1, class _Ty2>
-struct _Swap_cannot_throw : bool_constant<noexcept(swap(_STD declval<_Ty1>(), _STD declval<_Ty2>()))
-                                          && noexcept(swap(_STD declval<_Ty2>(), _STD declval<_Ty1>()))> {
+struct _Swap_cannot_throw : bool_constant<noexcept(swap(_STD declval<_Ty1>(), _STD declval<_Ty2>()))&& noexcept(

```

Comment:
Ugh, this is a regression in appearance. Add an empty comment to force wrapping before the logical AND?